SUPPORT.md

```diff
@@ -0,0 +1,34 @@
+## How do I&hellip;?
```

Comment:
HTML-fu for ...

![image](https://user-images.githubusercontent.com/247917/28569664-7deaefe6-7143-11e7-810e-fdcab1d45311.png)